metrotranscom/doorway
# feat: add data viz banner

This PR addresses #(insert-number-here)


## Description

It adds data viz banner, and align `Sign up for listing alerts` banner with design.

## How Can This Be Tested/Reviewed?

add some url (with protocol) as `MTC_DATA_URL` to public .env file.
it will show banner on homepage.
Also add `NOTIFICATIONS_SIGN_UP_URL` so you can test if already existing banner works as expected (it will be right above other banner).

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
